src/main/java/dan200/computercraft/shared/turtle/core/TurtleBrain.java

```diff
@@ -44,7 +44,7 @@
 
 public class TurtleBrain implements ITurtleAccess
 {
-    private static final int ANIM_DURATION = 8;
+    private static final int ANIM_DURATION = ComputerCraft.turtleAnimationDuration;
```

Comment:
If `ANIM_DURATION` is replaced everywhere else by `ComputerCraft.turtleAnimationDuration`, why not simply remove this field?

Reply:
This is why it's a draft, I new I wasn't finished.